Source/OxyPlot/Series/LineLegendPosition.cs

```diff
@@ -24,9 +24,33 @@ public enum LineLegendPosition
         /// </summary>
         Start,
 
+        /// <summary>
+        /// Render legend at the start of the line inside the plot area.
+        /// </summary>
+        StartInside,
+
+        /// <summary>
+        /// Render legend above the midpoint of the line inside the plot area.
+        /// </summary>
+        HalfWayAbove,
+
+        /// <summary>
+        /// Render legend above the midpoint of the line inside the plot area. This will follow the slope of the line
+        /// </summary>
+        HalfWayAboveSloped,
+
+        /// <summary>
+        /// Render legend below the midpoint of the line inside the plot area.
+        /// </summary>
+        HalfWayBelow,
+
+        /// <summary>
+        /// Render legend below the midpoint of the line inside the plot area. This will follow the slope of the line.
+        /// </summary>
+        HalfWayBelowSloped,
         /// <summary>
         /// Render legend at the end of the line.
         /// </summary>
         End
```

Comment:
Binary breaking change (change of constant)

Reply:
I've moved the "End" to the third position in the enumerator. Is this the change you had in mind?